_about/groups/ariawg.md

```diff
@@ -4,5 +4,94 @@ nav_title: '<span>ARIA: Accessible Rich Internet Applications <abbr title="Worki
 order: 4
 group: groups
 permalink: /about/groups/ariawg/
-redirect_to: https://www.w3.org/WAI/ARIA/
----
\ No newline at end of file
+---
+
+{::nomarkdown}
+{% include box.html type="start" title="Summary" class="" %}
+{:/}
+
+The mission of the Accessible Rich Internet Applications Working Group (ARIA WG) is to develop technologies that enhance accessibility of web content for people with disabilities. This includes continued development of the Accessible Rich Internet Applications (WAI-ARIA) suite of technologies and other technical specifications when needed to bridge known gaps.
+
+Some information on this page is also shown on, and may be more current in, the automatically generated [ARIA Working Group page](https://www.w3.org/groups/wg/aria/).
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+{::options toc_levels="2,3,4" /}
+
+{::nomarkdown}
+{% include_cached toc.html type="start" title="Page Contents" %}
+{:/}
+
+-   TOC is created automatically.
+{:toc}
+
+{::nomarkdown}
+{% include_cached toc.html type="end" %}
+{:/}
+
+## Announcements
+
+No announcements at the moment.
+
+## Current Work
+
+The Working Group has established a [road map](roadmap) for upcoming work. This road map is executed via [publications and status](deliverables), which include:
+
+- WAI-ARIA 1.3,
+- WAI-ARIA modules for Digital Publishing and Graphics,
+- Accessibility API Mappings for ARIA, Digital Publishing, HTML, and SVG, and
+- WAI-ARIA Authoring Practices.
+
+The [ARIA Project Plan](project) details intended timeline and milestones for this work and proceeds according to the [ARIA Working Group Work Flow](workflow).
+
+## Task Forces
+
+The ARIA WG uses [task forces](task-forces) to focus work on specific projects. Current task forces include:
+
+- ARIA Authoring Practices Task Force: focuses on ARIA Authoring Practices.
+- CSS Accessibility Task Force; focuses on accessibility of CSS, ARIA role is to explore ARIA and accessibility API mapping features.
+
+## How to Comment, Contribute, and Participate
+
+The ARIA Working Group engages with stakeholders in a variety of ways. See the following resources for information on:
+
+- [How to contribute to the Working Group, file comments, and contribute to the source code repositories](contribute);
+- [How to participate in (join) the Working Group](https://www.w3.org/groups/wg/aria/instructions/).
+
+## Meetings and Communication
+
+The ARIA WG conducts its work using a variety of synchronous and asynchronous tools. The [communication](aria/communication) page provides details about:
+
+- Teleconferences of the Working Group and its task forces (also see [meeting minutes](aria/minutes));
+- Face to face meetings (also see face to face [meeting pages](https://www.w3.org/WAI/ARIA/wiki/Meetings));
```

Comment:
Again, I think let's keep this link as-is until we decide what to do with the minutes tool.;zzz